### DISCUSSION TITLE
randomly not starting new jobs

### DISCUSSION THREAD

**Discussion by alexb373** [swar/Swar-Chia-Plot-Manager#346] [Q&A]
Have Swar Manager installed on 4 Windows 10 machines. When checking progress sometimes see that a new job hasn't started, but supposed to according to the config.

A simple stop/start manager resolves the issue (additional jobs are being launched without editing config or anything else).

This happened multiple times on different machines, no idea why.

What could be causing this? How to avoid the issue?

**Comment by Arkex-3** [comment]
same issue for me with plots

can't even get the manager back to work with start/stop/restart
even if im adding another job plots won't start

**Comment by swar** [comment]
Try running it as admin. Please use Github Discussions next time.

**Comment by Lyushen** [comment]
post your configs here for exam it
